Detectors/TOF/calibration/macros/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
+# under the terms of the GNU General Public License v3 (GPL Version 3), copied
+# verbatim in the file "COPYING".
+#
+# See http://alice-o2.web.cern.ch/license for full licensing information.
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+if(BUILD_TESTING)
+  o2_add_test(
```

Comment:
I actually did not want to test this, but it was complaining if I did not. 
Should I simply add it to the exceptions?

Reply:
if you wanted to simply have an executable, then use ``o2_add_executable`` instead of ``o2_add_test``. Also, you probably don't need ``if(BUILD_TESTING)``

Reply:
done!